### DISCUSSION TITLE
Duplicate entries in entry fields

### DISCUSSION THREAD

**Discussion by mihob** [craftcms/cms#13851] [Q&A] [answered]
Is it somehow possible to select entries multiple times in entry fields?

I have a content module that should display teasers. Teasers are created as entries to avoid having to maintain them multiple times.

Now these teaser lists can be quite long and especially important entries should appear several times. Is this somehow feasible?

**Comment by piotrpog** [comment, marked as answer]
Alternative way to do thus would be to create matrix fields where each block has entry field. That way you can have multiple blocks with same entry selected.

**Reply by brandonkelly** [reply]
This is probably the closest you’re going to get.

**Reply by mihob** [reply]
Yes thanks! Have already solved it this way

**Comment by brandonkelly** [comment]
It’s not possible, nor would it be possible to do it with a custom field type, as there’s a unique constraint on the `relations` table on the `fieldId`, `sourceId`, `sourceSiteId`, and `targetId` columns.
